### PR TITLE
[V7-2846] Add ndpi support

### PR DESCRIPTION
**Pull request by simedw**

### CODE REVIEW

**Comment by Fl4m3Ph03n1x** [comment]
I have no idea what `ndpi` nor what is used for. 
Based on the code I guess it is some kind of video. 

Looks good to me.

**Review by Fl4m3Ph03n1x — APPROVED** [review]
If it passes the pipeline, its good to go.

**Review by andreaazzini — APPROVED** [review]

**Comment by Fl4m3Ph03n1x** [comment]
Cannot merge until pipeline is fixed.

File: darwin/dataset/split_manager.py [hunk 1]
```diff
@@ -110,7 +110,8 @@ def split_dataset(
     # Requirements: scikit-learn
     try:
         import sklearn  # noqa
-    except ImportError:
+    except ImportError as error:
+        print(error)
```
Comment (andreaazzini) [inline darwin/dataset/split_manager.py:114]: Is this expected?

**Review by andreaazzini — CHANGES_REQUESTED** [review]

File: darwin/dataset/split_manager.py (hunk 1 above)
Comment (simedw) [reply darwin/dataset/split_manager.py:114]: this is pedro debugging the issue

**Review by Fl4m3Ph03n1x — APPROVED** [review]

**Review by andreaazzini — APPROVED** [review]

**Pull request merged**